pom.xml

```diff
@@ -194,21 +191,42 @@
 
 	<repositories>
 		<repository>
-			<id>spring-releases</id>
-			<url>https://repo.spring.io/libs-release</url>
+			<id>spring-snapshots</id>
```

Comment:
We really shouldn't be using snapshots because it makes the code unpredictable from one run to another (or my computer vs your computer).